modules/openapi-generator/src/main/resources/Java/libraries/feign/auth/ApiErrorDecoder.mustache

```diff
@@ -18,7 +20,7 @@ public class ApiErrorDecoder implements ErrorDecoder {
         Exception httpException = defaultErrorDecoder.decode(methodKey, response);
         if (response.status() == 401 || response.status() == 403) {
             return new RetryableException(response.status(), "Received status " + response.status() + " trying to renew access token",
-                    response.request().httpMethod(), httpException, null, response.request());
+                    response.request().httpMethod(), httpException, (Date) null, response.request());
```

Comment:
I don't understand why you need to cast it to Date given that it's null?

Reply:
In feign RetryableException now implement two constructor with the same argument except for the 5th who can now be a Date (Like previous version of feign) or a Long. 
Then when you put null for the 5th argument the compiler don't know if you want to use the Constructor with Date or Long and the result is an exception with the compiler telling you their an ambiguity.
When their is no ambiguity the compiler consider by himself that null is a Date. 

In that case I think using a Date or a Long lead to the same result. But sometime it can lead to different behaviour this is why jave compiler fail.

Reply:
I ran `mvn integration-test -f samples/client/petstore/java/feign/pom.xml` which completed successfully without issue.

Did you get any errors when running the above command at the root level of this project?

Are you using a different version of feign?

Reply:
i've used gitpod feign was ok but feign-no-nullable was ko

Reply:
so you set the option `openApiNullable` to false for your use cases?

Reply:
No i Have just run all integration. 
https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/java/feign is ok with my pr. 

but https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/java/feign-no-nullable is ko with and without my pr

Reply:
can you please PM me via Slack when you've time?

https://join.slack.com/t/openapi-generator/shared_invite/zt-12jxxd7p2-XUeQM~4pzsU9x~eGLQqX2g

Reply:
Our build fails with any 13.x version of Feign because of this.